SFS_config.m

```diff
@@ -468,17 +473,34 @@
 
 %% ===== References ======================================================
 %
-% Spors, Rabenstein, Ahrens - The Theory of Wave Field Synthesis Revisited, 124
-% AES Convention, Paper 7358, 2008. http://bit.ly/ZCvyQ6
+% Ahrens (2012) - "Analytic Methods of Sound Field Synthesis", Springer,
+% https://doi.org/10.1007/978-3-642-25743-8
+%
+% Ahrens, Spors (2010) - "Sound Field Reproduction Using Planar and Linear
+% Arrays of Loudspeakers", Transactions on Audio, Speech, and Language
+% Processing, vol. 18, no. 8, pp. 2038-2050,
+% https://doi.org/10.1109/TASL.2010.2041106
+%
+% Born, Wolf (1999) - "Principles of Optics", Cambridge University Press, 7th
```

Comment:
according to http://opac.lbs-rostock.gbv.de/DB=1/XMLPRS=N/PPN?PPN=559382774 , the 7th Ed. is from 2006. Either the edition or the year is wrong?